src/main/java/org/walkmod/javalang/compiler/types/IOUtil.java

```diff
@@ -0,0 +1,30 @@
+package org.walkmod.javalang.compiler.types;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class IOUtil {
+
+  public static byte[] readStream(InputStream inputStream, boolean close) throws IOException {
+    if(inputStream == null) {
+      throw new IOException("Stream not found");
```

Comment:
I wasted too much time finding the real cause because of unhelpful exceptions like this. Pass and add resource name?